weaveworks/weave-gitops-enterprise
# Import weave-gitops npm module!

- And try and make it work!

To test:
- Run wge-ui
  - Make sure you have a `GITHUB_TOKEN` in your bashrc / bash env that has read_packages scope, update token scope here https://github.com/settings/tokens otherwise you'll get a `yarn install` error about auth.
  - `cd weave-gitops-enterprise/ui-cra && yarn`
  - `yarn start`
